plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -333,6 +344,23 @@ class NewCreateServiceModalForm extends Component {
             onRemoveItem={this.handleRemoveItem}
             path={`containers.${index}`}
             service={service} />
+
+          <AdvancedSection>
+            <AdvancedSectionLabel>
+              More Settings
+            </AdvancedSectionLabel>
+            <AdvancedSectionContent>
+              <h3 className="short-top short-bottom">
+                Advanced Settings
+              </h3>
+              <ArtifactsSection
```

Comment:
Purely visual #nit ..  it kind of looks bad how the headings are stuck to each other. Can you ad an one-line description paragraph under the "Advanced Settings"? Something stupidly obvious, such as "Advanced options for the container" just to space things a bit 😄 

![image](https://cloud.githubusercontent.com/assets/883486/23758540/6b77d13c-04ea-11e7-898f-2f651e6f341f.png)